### PR TITLE
Example notebook for classification + dry_run flag draft

### PR DESCRIPTION
**Pull request by twinbrian**
## Description
Example notebook of classification in regards to Issue #1030, pretty much similar to workflow for regression but with some changes in normalization and FFN.

## Questions
Since I ran tests on my m2, I used accelerator as cpu due to low MPS ops coverage, but probably could be changed to "auto"

### CODE REVIEW

File: examples/training_classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,909 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "chemprop_dir = Path.cwd().parent\n",
+    "input_path = chemprop_dir / \"tests\" / \"data\" / \"classification\" / \"mol.csv\" # path to your data .csv file\n",
+    "num_workers = 7 # number of workers for dataloader. 0 means using main process for data loading\n",
```
Comment (shihchengli) [inline examples/training_classification.ipynb:46]:
```suggestion
    "num_workers = 0 # number of workers for dataloader. 0 means using main process for data loading\n",
```

File: examples/training_classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,909 @@
...
+   "source": [
+    "df_input = pd.read_csv(input_path)\n",
+    "df_input"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
```
Comment (shihchengli) [inline examples/training_classification.ipynb:229]:
```suggestion
   "cell_type": "markdown",
   "metadata": {},
```

**Comment by jonwzheng** [comment]
We're trying to figure out why the training cell has an output for each epoch which is in contrast to the other notebooks. I reran it on Ubuntu, but it doesn't seem to fix the issue.

**Comment by KnathanM** [comment]
Hi @twinbrian, I opened a PR against your branch on your repo. https://github.com/twinbrian/chemprop/pull/1

**Comment by twinbrian** [comment]
opened two new PR #1047 and #1048

**Pull request closed without merging**